testproject/Packages/manifest.json

```diff
@@ -1,48 +1,47 @@
-      "com.unity.modules.physics": "1.0.0",
-      "com.unity.modules.physics2d": "1.0.0",
-      "com.unity.modules.screencapture": "1.0.0",
-      "com.unity.modules.terrain": "1.0.0",
-      "com.unity.modules.terrainphysics": "1.0.0",
-      "com.unity.modules.tilemap": "1.0.0",
-      "com.unity.modules.ui": "1.0.0",
-      "com.unity.modules.uielements": "1.0.0",
-      "com.unity.modules.umbra": "1.0.0",
-      "com.unity.modules.unityanalytics": "1.0.0",
-      "com.unity.modules.unitywebrequest": "1.0.0",
-      "com.unity.modules.unitywebrequestassetbundle": "1.0.0",
-      "com.unity.modules.unitywebrequestaudio": "1.0.0",
-      "com.unity.modules.unitywebrequesttexture": "1.0.0",
-      "com.unity.modules.unitywebrequestwww": "1.0.0",
-      "com.unity.modules.vehicles": "1.0.0",
-      "com.unity.modules.video": "1.0.0",
-      "com.unity.modules.vr": "1.0.0",
-      "com.unity.modules.wind": "1.0.0",
-      "com.unity.modules.xr": "1.0.0"
-    },
-    "testables": [
-      "com.unity.multiplayer.mlapi"
-    ]
-  }
-  
\ No newline at end of file
+  "dependencies": {
+    "com.unity.collab-proxy": "1.2.16",
+    "com.unity.ide.rider": "1.1.4",
+    "com.unity.ide.vscode": "1.2.3",
+    "com.unity.multiplayer.mlapi": "file:../../com.unity.multiplayer.mlapi",
+    "com.unity.package-validation-suite": "0.19.2-preview",
+    "com.unity.test-framework": "1.1.22",
+    "com.unity.textmeshpro": "2.1.1",
+    "com.unity.timeline": "1.2.17",
+    "com.unity.ugui": "1.0.0",
+    "com.unity.modules.ai": "1.0.0",
+    "com.unity.modules.androidjni": "1.0.0",
+    "com.unity.modules.animation": "1.0.0",
+    "com.unity.modules.assetbundle": "1.0.0",
+    "com.unity.modules.audio": "1.0.0",
+    "com.unity.modules.cloth": "1.0.0",
+    "com.unity.modules.director": "1.0.0",
+    "com.unity.modules.imageconversion": "1.0.0",
+    "com.unity.modules.imgui": "1.0.0",
+    "com.unity.modules.jsonserialize": "1.0.0",
+    "com.unity.modules.particlesystem": "1.0.0",
+    "com.unity.modules.physics": "1.0.0",
+    "com.unity.modules.physics2d": "1.0.0",
+    "com.unity.modules.screencapture": "1.0.0",
+    "com.unity.modules.terrain": "1.0.0",
+    "com.unity.modules.terrainphysics": "1.0.0",
+    "com.unity.modules.tilemap": "1.0.0",
+    "com.unity.modules.ui": "1.0.0",
+    "com.unity.modules.uielements": "1.0.0",
+    "com.unity.modules.umbra": "1.0.0",
+    "com.unity.modules.unityanalytics": "1.0.0",
+    "com.unity.modules.unitywebrequest": "1.0.0",
+    "com.unity.modules.unitywebrequestassetbundle": "1.0.0",
+    "com.unity.modules.unitywebrequestaudio": "1.0.0",
+    "com.unity.modules.unitywebrequesttexture": "1.0.0",
+    "com.unity.modules.unitywebrequestwww": "1.0.0",
+    "com.unity.modules.vehicles": "1.0.0",
+    "com.unity.modules.video": "1.0.0",
+    "com.unity.modules.vr": "1.0.0",
+    "com.unity.modules.wind": "1.0.0",
+    "com.unity.modules.xr": "1.0.0"
+  },
+  "testables": [
+    "com.unity.multiplayer.mlapi"
+  ]
+}
```

Comment:
(oh no)

Reply:
So, after looking at this , I realized the only change here is the upgrade to the "com.unity.test-framework": "1.1.22" which is what the default package for 2019.4.20f1 LTS.  So, depending upon the answer about the 2019.4.20f1 LTS and the 2019.4.17f1(non-LTS version) this might actually be a good thing that it was changed.